File: KremenchukZdo70Site/ClientApp/src/app/shared/models/employee-request.ts
import { JobTitleItemResponse } from './job-title-item-response';

export class EmployeeRequest {
  id: number = 0;
  firstName: string | undefined;
  secondName: string | undefined;
  lastName: string | undefined;
  smallProfileUrl: string | undefined;
  fullProfileUrl: string | undefined;
}

File: KremenchukZdo70Site.Domain/Request/EmployeeRequest.cs
namespace KremenchukZdo70Site.Domain.Request
{
    public class EmployeeRequest
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string SecondName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string SmallProfileUrl { get; set; } = string.Empty;
        public string FullProfileUrl { get; set; } = string.Empty;
    }
}
